python/ambassador_diag/diagd.py

```diff
@@ -460,10 +463,23 @@ def show_overview(reqid=None):
 
     ddict = collect_errors_and_notices(request, reqid, "overview", diag)
 
+    banner_content = None
+    app.logger.info(app.banner_endpoint)
+    if app.banner_endpoint:
+        try:
+            response = requests.get(app.banner_endpoint)
+            app.logger.info(response)
+            if response.status_code == 200:
+                banner_content = response.text
+        except Exception as e:
+            app.logger.error("could not get banner_content: %s" % e)
+            app.logger.exception(e)
```

Comment:
Some of the other things (like `envoy_stats`) are polled periodically rather than at every call to the overview. Would that make more sense? or would a push make sense?

Also, should the detail view show this as well?

Reply:
No polling for now, as we expect the banner content might get more dynamic over time, and only for the overview.